.travis.yml

```diff
@@ -8,6 +8,7 @@ android:
   - tools
 
   - build-tools-25.0.0
+  - android-25
   - android-24
```

Comment:
Yes; one to compile against and another for the emulator. There's no emulator image for android-25 for some reason.